core/services/postgres/transaction.go

```diff
@@ -89,3 +90,44 @@ func DBWithDefaultContext(db *gorm.DB, fc func(db *gorm.DB) error) error {
 	defer cancel()
 	return fc(db.WithContext(ctx))
 }
+
+func WrapDbWithSqlx(rdb *sql.DB) *sqlx.DB {
+	db := sqlx.NewDb(rdb, "postgres")
+	db.MapperFunc(CamelToSnakeASCII)
```

Comment:
What's wrong with simply using CamelCase in the sql strings?

Reply:
It works on inserts and queries, but it wasn't able to map snake_case column names to struct fields when struct scanning.